Dockerfile

```diff
@@ -34,25 +23,18 @@ RUN wget --directory-prefix=${TEMP_DIR} ${OC_DL_URL} && \
     tar -C ${OC_BIN} -xzf ${TEMP_DIR}/${OC_BIN_TAR} && \
     chmod a+x ${OC_BIN}/oc
 
-# Add go and oc binary directory to $PATH
-ENV PATH=${PATH}:"/usr/local/go/bin":${GOPATH}/"bin"
-
-# Git identifier to checkout
-ARG TNF_VERSION
-ARG TNF_SRC_URL=$TNF_SRC_URL
-ARG GIT_CHECKOUT_TARGET=$TNF_VERSION
-
-# Clone the TNF source repository and checkout the target branch/tag/commit
-RUN git clone --no-single-branch --depth=1 ${TNF_SRC_URL} ${TNF_SRC_DIR}
-RUN git -C ${TNF_SRC_DIR} fetch origin ${GIT_CHECKOUT_TARGET}
-RUN git -C ${TNF_SRC_DIR} checkout ${GIT_CHECKOUT_TARGET}
+# Download operator-sdk binary
+ENV OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.26.0
+ENV OSDK_BIN="/usr/local/osdk/bin"
+RUN mkdir -p ${OSDK_BIN} && \
+    curl -LO ${OPERATOR_SDK_DL_URL}/operator-sdk_linux_amd64 && \
+    mv operator-sdk_linux_amd64 ${OSDK_BIN}/operator-sdk && \
+    chmod +x ${OSDK_BIN}/operator-sdk
 
-# Build TNF binary
+# Copy all of the files into the source directory and then switch contexts
+COPY . ${TNF_SRC_DIR}
```

Comment:
Another optimization could be to remove all the source code from the final TNF image, as I don't it's needed. Maybe keeping this `TNF_SRC_DIR` separeted from `TNF_DIR` is a good way to avoid copying it.

Reply:
Maybe this can be an optimization in a follow up PR?

Reply:
Sure, no problem.